.changeset/deprecate-actionlistv1-promote-actionlistv2.md

```diff
@@ -0,0 +1,154 @@
+---
+'@primer/react': major
+---
+
+### ActionList
```

Comment:
I think we might need a linebreak here

![CleanShot 2022-02-28 at 14 03 01](https://user-images.githubusercontent.com/4608155/156065962-01d69147-f5b0-46ed-b0a4-9fa139fa632f.png)



```suggestion
<br />

### ActionList
```